attractions/file-input/file-dropzone.svelte

```diff
@@ -13,6 +13,7 @@
   export let files = [];
   export let fileComponent = FileTile;
   export let accept = null;
+  export let beforeChange = null;
```

Comment:
To avoid the sanity check in the `acceptUpload` function, it might be better to use a no-op function as the default
```suggestion
  export let beforeChange = async () => {};
```

Reply:
Nah